ocaml/xapi/xapi.ml

```diff
@@ -189,6 +189,7 @@ let init_args() =
   (* Immediately register callback functions *)
   register_callback_fns();
   Xcp_service.configure ~options:Xapi_globs.all_options ~resources:Xapi_globs.Resources.xcp_resources ();
+  Xcp_coverage.dispatcher_init "xapi";
```

Comment:
How did you decide where in the start-up sequence to insert this?
There are many places that look as if they might be reasonable, but it is quite possible that some have non-obvious drawbacks.

Reply:
The other daemons (well xenopsd for now) would use `Xcp_service.configure2` to initialize, so I chose the most similar place to that, there are indeed many equally good places for it.
I think the initialization should meet these constraints:
* we initialize before shutting down in a successful run (so the coverage data gets written in the proper place)
* we do not initialize it for the CLI (we would run out of space in the host-installer ramdisk)
* we initialize before a testcase is finished (which triggers the coverage dump call)